aboutme.html

```diff
@@ -12,23 +12,20 @@
     <a class="navbar__link" href="aboutme.html">About Me</a>
     <a class="navbar__link" href="projects.html">Projects</a>
     <a class="navbar__link" href="education.html">Education</a>
-    <a class="navbar__link" href="#contact">Contact</a>
+    <a class="navbar__link" href="contact.html">Contact</a>
     </nav>
 </header>
     
 <main class="hero">
   <h1 class="hero__title"> About Me</h1>
   <h2 class="hero__subtitle">Let me introduce myself!</h2>
 </main>
-    
+
 <div class="content">
-  <img class="content__img" src="https://scontent.fphl2-1.fna.fbcdn.net/v/t31.0-8/20643305_1943469985936178_8331495383796159137_o.jpg?_nc_cat=100&_nc_oc=AQncJ-6wwbt1qo3bXBKHj_j9DR03uP_uDLoIjNq-3-nqeJ3y_A-K196lX8K8vcYRLCs&_nc_ht=scontent.fphl2-1.fna&oh=ad1da48ead3f229adcc8830304bc3205&oe=5DFC5CA9">
-  <p class="content__copy">Hello! This is my portfolio, and this is Nicole.</p><br>
-  <p class="content__copy">I am an aspiring web developer that has always loved learning about "the process."  The process of creating a thrown pot, a well crafted latte, a flowing story (consequentially I have been an ameteur artist, barista, and English major).  Unsuprisingly I developed a passion for learning about one of the most ever present objects in our contemporary lives, the web- and what appears on it.</p><br>
-<p class="content__copy">Everything is brought together by elements that play off of each other and are ordered in a way to create one specific outcome.  Learning how to program has been a new form of creation that I am very excited to further pursue and master.  I hope you enjoy my portfolio here as it is and see that future with me! Feel free to explore!</p>
+<div class="aboutme">
```

Comment:
`aboutme` is two words so recommended format would be `about-me__foo` or `aboutMe__foo`